modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/util/ObjectRelationshipTestUtil.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.related.models.test.util;
+
+import com.liferay.object.model.ObjectEntry;
+import com.liferay.object.model.ObjectRelationship;
+import com.liferay.object.related.models.ObjectRelatedModelsProvider;
+import com.liferay.object.service.ObjectRelationshipLocalService;
+import com.liferay.portal.kernel.dao.orm.QueryUtil;
+import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import org.junit.Assert;
+
+/**
+ * @author Pedro Leite
+ */
+public class ObjectRelationshipTestUtil {
+
+	public static void addObjectRelationshipMappingTableValues(
+			long objectRelationshipId,
+			ObjectRelationshipLocalService objectRelationshipLocalService,
+			long primaryKey1, long primaryKey2)
+		throws Exception {
+
+		objectRelationshipLocalService.addObjectRelationshipMappingTableValues(
+			TestPropsValues.getUserId(), objectRelationshipId, primaryKey1,
+			primaryKey2, ServiceContextTestUtil.getServiceContext());
+	}
+
+	public static void assertGetRelatedModels(
+			int expectedSize,
+			ObjectRelatedModelsProvider objectRelatedModelsProvider,
+			long objectRelationshipId, long primaryKey)
+		throws Exception {
+
+		List<ObjectEntry> objectEntries =
+			objectRelatedModelsProvider.getRelatedModels(
+				0, objectRelationshipId, primaryKey, null, QueryUtil.ALL_POS,
+				QueryUtil.ALL_POS);
+
+		Assert.assertEquals(
+			objectEntries.toString(), expectedSize, objectEntries.size());
+	}
+
+	public static void assertSearchRelatedModels(
+			int expectedSize,
+			ObjectRelatedModelsProvider objectRelatedModelsProvider,
+			long objectRelationshipId, long primaryKey, String search)
+		throws Exception {
+
+		List<ObjectEntry> objectEntries =
+			objectRelatedModelsProvider.getRelatedModels(
+				0, objectRelationshipId, primaryKey, search, QueryUtil.ALL_POS,
+				QueryUtil.ALL_POS);
+
+		Assert.assertEquals(
+			objectEntries.toString(), expectedSize, objectEntries.size());
+	}
+
+	public static ObjectRelationship updateObjectRelationship(
+			String deletionType, long objectRelationshipId,
+			Map<Locale, String> objectRelationshipLabelMap,
+			ObjectRelationshipLocalService objectRelationshipLocalService)
+		throws Exception {
+
+		return objectRelationshipLocalService.updateObjectRelationship(
+			objectRelationshipId, 0, deletionType, objectRelationshipLabelMap);
```

Comment:
@pedro-oliveira446 Let's simplify and use the `ObjectRelationshipLocalServiceUtil`. 

In addition, please sort and rename based on the service builder, i.e., `long objectRelationshipId, String deletionType, Map<Locale, String> labelMap`